proto/substrait/algebra.proto

```diff
@@ -589,6 +591,13 @@ message WriteRel {
     WRITE_OP_CTAS = 4;
   }
 
+  enum CreateMode {
+    CREATE_MODE_UNSPECIFIED = 0;
+    CREATE_MODE_ERROR_IF_EXISTS = 1; // Throw an error if the table already exists (default behavior)
+    CREATE_MODE_REPLACE_IF_EXISTS = 2; // Replace the table if it already exists ("OR REPLACE")
```

Comment:
Is there a way to separate the write behavior from the error handling?  I don't have a clear implementation in mind but where each of these three settings (WriteOp, CreateMode, OutputMode) have one dimension of behavior that they deal with.

Reply:
I feel this is easier to follow as it is closer to the sql.